crates/miden-tx/src/executor/exec_host.rs

```diff
@@ -109,18 +110,34 @@ where
     ) -> Result<(), TransactionKernelError> {
         let pub_key = process.get_stack_word(0);
         let msg = process.get_stack_word(1);
+
         let signature_key = Hasher::merge(&[pub_key, msg]);
 
         let signature = if let Ok(signature) =
             process.advice_provider().get_mapped_values(&signature_key)
         {
             signature.to_vec()
         } else {
-            let signing_inputs = SigningInputs::Blind(msg);
+            let commitments = process
+                .advice_provider()
+                .get_mapped_values(&msg)
+                .map_err(|err| TransactionKernelError::SignatureGenerationFailed(Box::new(err)))?;
```

Comment:
I wonder if we should add a specialized error variant for this to make it clear that transaction summary commitments were not found in the advice provider.